### PR TITLE
[Adding file1-4.txt] - test pull request

### PR DESCRIPTION
**Pull request by jbossops**
[Adding file1-4.txt] - test pull request

### CODE REVIEW

File: branch1/file1-4.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Lorem ipsum
```
Comment (LinusTorval) [inline branch1/file1-4.txt:1]: OK

File: branch2/file2-2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+lorem ipsum
```
Comment (LinusTorval) [inline branch2/file2-2.txt:1]: OK

File: branch2/file2.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+file2
```
Comment (LinusTorval) [inline branch2/file2.txt:1]: OK

File: branch1/file1-4.txt (hunk 1 above)
Comment (jbossops) [reply branch1/file1-4.txt:1]: Please merge it

File: branch2/file2-2.txt (hunk 2 above)
Comment (jbossops) [reply branch2/file2-2.txt:1]: Please merge it

File: branch2/file2.txt (hunk 3 above)
Comment (jbossops) [reply branch2/file2.txt:1]: Please merge it

**Pull request merged**